python/pylibvw.cc

```diff
@@ -545,15 +545,14 @@ void ex_push_feature(example_ptr ec, unsigned char ns, uint32_t fid, float v)
 {  // warning: assumes namespace exists!
   ec->feature_space[ns].push_back(v, fid);
   ec->num_features++;
-  ec->total_sum_feat_sq += v * v;
+  ec->reset_total_sum_feat_sq();
```

Comment:
should this be resetting? it looks like it was adding previously

Reply:
This means to reset any cached calculation. Since this value is rarely but (I think is reused when calculated) I store a memoized copy of the value. Since the old code changes the features it changes what the output would be

Reply:
ok makes sense